syntax = "proto3";

package FLogProto;

// The service definition.
service FLogRemoteLogger {
  rpc SendLogLine (LogLine) returns (Response) {}
}

message LogLine {
  bytes log = 1;
}

message Response {
  bytes message = 1;
}
